### DISCUSSION TITLE
it's a hassle

### DISCUSSION THREAD

**Discussion by Sadoktns** [community/community#67391] [Programming Help]
### Select Topic Area

General

### Body

hi, i'm a newbie programer, and i have a big problem with a english so sorry, i degin with a html program before i test python but i not apprecie it because i have many problem with it so i rage quit. i have a question about my html program.  i did a a website about a weather who give a temperature. So my question is how do a functional search. thanks if you help me

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Sadoktns ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
